src/bridge/IrcBridge.ts

```diff
@@ -1337,8 +1340,8 @@ export class IrcBridge {
     }
 
     public async connectToIrcNetworks(): Promise<void> {
-        await promiseutil.allSettled(this.ircServers.map((server) =>
-            Bluebird.cast(this.clientPool.loginToServer(server))
+        await Promise.all(this.ircServers.map((server) =>
```

Comment:
We actually want to throw if this fails, otherwise we'd be running the IRC bridge with a possible configuration issue (if the bots cant connect).